src/api/routes/v1/amendments.ts

```diff
@@ -0,0 +1,110 @@
+import { Request, Response } from 'express'
```

Comment:
Is the API stuff supposed to be in a different PR?

Reply:
This would just be for the `amendment/info` endpoint. The other PRs would add to validators and `amendment/vote` endpoints